### PR TITLE
Update Drupal Readme

### PR DESCRIPTION
**Pull request by TChukwuleta**
This is part work on https://github.com/btcpayserver/btcpayserver-doc/issues/1352  updating Drupal documentation
 
 Cc. @ndeet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
-# Commerce BTCPay
+# Accept Bitcoin on your Drupal Store using BTCPay Server
```
Comment (ndeet) [inline README.md]: There are different Drupal ecommerce solution, the one my plugin is for is called "Drupal Commerce" so I would suggest to change "Drupal Store" to "Drupal Commerce store"

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
-This module provides a [Drupal Commerce 2.x](https://www.drupal.org/project/commerce) payment plugin for [BTCPay Server](https://docs.btcpayserver.org). This allows you to accept cryptocurrencies without a 3rd party intermediary by becoming your own payment processor.
+Introducing BTCPay Server payment plugin for [Drupal Commerce 2.x](https://www.drupal.org/project/commerce). Drupal Store owners can now accept payments using Bitcoin and other cryptocurrencies directly through BTCPay Server without any third-party intermediary.
```
Comment (ndeet) [inline README.md]: In Drupal ecosystem, extensions are called "modules" and not plugins.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 
-Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
+BTCPay Server supports a wide range of cryptocurrencies, with the potential for future extensions. Here are the currencies you can use now on BTCPay Server:
+
+- BTC (Bitcoin)
+- Bitcoin 2nd later lightning network for fast and zero/low-fee transactions
```
Comment (ndeet) [inline README.md]: typo: later -> layer Lightning Network

File: README.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+- Altcoins with full node integration including coins like Monero (XMR) and Litecoin (LTC).
+- Other Major Altcoins: Supported through plugins via platforms such as [Trocador](https://docs.btcpayserver.org/Trocador/), [SideShift](https://docs.btcpayserver.org/SideShift/), and FixedFloat.
+
+Want to accept Bitcoin on your drupal store? Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
```
Comment (ndeet) [inline README.md]: ... Drupal Commerce store

File: README.md [hunk 5]
```diff
@@ -14,60 +21,54 @@ A Drupal Commerce demo store connected with a (testnet) BTCPay Server where you
 * Drupal Commerce 2.x installed ([installation guide](https://docs.drupalcommerce.org/commerce2/developer-guide/install-update/installation))  
 * Drupal: [configured and writable private file system](https://www.drupal.org/docs/8/core/modules/file/overview#content-accessing-private-files)
 
-## Installation and configuration
+## Installation and configuration Guide for BTCPay Server-Drupal Commerce Integration
```
Comment (ndeet) [inline README.md]: Installation and configuration guide for the BTCPay Server - Drupal Commerce integration

File: README.md [hunk 6]
```diff
@@ -14,60 +21,54 @@ A Drupal Commerce demo store connected with a (testnet) BTCPay Server where you
...
+
+Ready to accept Bitcoin on your Drupal Store? Follow this quick and easy guide to install and configure the BTCPay Drupal Commerce module. For a quick run through, check out our installation and configuration screencast:
```
Comment (ndeet) [inline README.md]: Drupal Commerce store

File: README.md [hunk 7]
```diff
@@ -14,60 +21,54 @@ A Drupal Commerce demo store connected with a (testnet) BTCPay Server where you
...
 
-### Quick walkthrough steps
+
+### Easy setup steps
 
 #### Generate pairing code on BTCPay server
-1.  BTCPay server: [create and configure a store](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-btcpay-store)
-2.  in store settings go to "**Access Tokens**"
-3.  click on **[Create a new token]**
+1.  **Setup your store:** You'd need a BTCPay server instance to get started. Don't have one? click [here](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-btcpay-store) for a step by step guide.
```
Comment (ndeet) [inline README.md]: This links to docs how to create a store but you suggest it shows how to create a BTCPay Server instance. Either change text to reflect store or change the link to the deployment options overview.

**Review by ndeet — COMMENTED** [review]
Thanks a lot Tobses and sorry for the late review, added a few suggestions and typo fixes.

**Comment by TChukwuleta** [comment]
Hey @ndeet  I have resolved the comments.

File: README.md [hunk 8]
```diff
@@ -1,8 +1,15 @@
...
 
-Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
+BTCPay Server supports a wide range of cryptocurrencies, with the potential for future extensions. Here are the currencies you can use now on BTCPay Server:
+
+- BTC (Bitcoin)
+- Bitcoin layer-two network (the lightning network) for fast and zero/low-fee transactions
+- Altcoins with full node integration including coins like Monero (XMR) and Litecoin (LTC).
+- Other Major Altcoins: Supported through plugins via platforms such as [Trocador](https://docs.btcpayserver.org/Trocador/), [SideShift](https://docs.btcpayserver.org/SideShift/), and FixedFloat.
+
+Want to accept Bitcoin on your drupal commerce store? Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
```
Comment (ndeet) [inline README.md]: nitpick: for consistency please "Drupal Commerce"

File: README.md [hunk 9]
```diff
@@ -1,8 +1,15 @@
...
+# Accept Bitcoin on your Drupal Commerce Store using BTCPay Server
 
-This module provides a [Drupal Commerce 2.x](https://www.drupal.org/project/commerce) payment plugin for [BTCPay Server](https://docs.btcpayserver.org). This allows you to accept cryptocurrencies without a 3rd party intermediary by becoming your own payment processor.
+Introducing BTCPay Server payment module for [Drupal Commerce 2.x](https://www.drupal.org/project/commerce). Drupal Commerce Store owners can now accept payments using Bitcoin and other cryptocurrencies directly through BTCPay Server without any third-party intermediary.
 
-Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
+BTCPay Server supports a wide range of cryptocurrencies, with the potential for future extensions. Here are the currencies you can use now on BTCPay Server:
+
+- BTC (Bitcoin)
+- Bitcoin layer-two network (the lightning network) for fast and zero/low-fee transactions
```
Comment (ndeet) [inline README.md]: nitpick: looks good but I meant more that you should have capital letters "Lightning Network"

**Review by ndeet — COMMENTED** [review]
few nitpicks :)

File: README.md (hunk 8 above)
Comment (TChukwuleta) [reply README.md]: all good

**Review by ndeet — APPROVED** [review]
thank you :partying_face:

**Pull request merged**